.misc/appveyor.yml

```diff
@@ -1,63 +1,92 @@
+image: Visual Studio 2015
```

Comment:
These changes were sync'd with the Fudge CI PR.

Please rebase and there should only be two files needing updating now.